<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DialogTypeJointure</class>
 <widget class="QDialog" name="DialogTypeJointure">
  <property name="windowModality">
   <enum>Qt::ApplicationModal</enum>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>246</width>
    <height>97</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Relation Type</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <normaloff>:/gapQBE.png</normaloff>:/gapQBE.png</iconset>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <widget class="QLabel" name="label">
       <property name="text">
        <string>Please choose a type:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="comboBoxType">
       <item>
        <property name="text">
         <string extracomment="Contrainte d'intégrité fonctionnelle">Cif</string>
        </property>
        <property name="icon">
         <iconset resource="mesressources.qrc">
          <normaloff>:/images/cif</normaloff>:/images/cif</iconset>
        </property>
       </item>
       <item>
        <property name="text">
         <string extracomment="Dépendance Faible">Df</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string extracomment="Patte d'association">Leg</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string extracomment="Entité faible">LeakRelation</string>
        </property>
        <property name="icon">
         <iconset>
          <normaloff>:/images/leakRelation.png</normaloff>:/images/leakRelation.png</iconset>
        </property>
       </item>
       <item>
        <property name="text">
         <string extracomment="Association">Relation</string>
        </property>
        <property name="icon">
         <iconset resource="mesressources.qrc">
          <normaloff>:/images/association</normaloff>:/images/association</iconset>
        </property>
       </item>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QLabel" name="labelRelationName">
       <property name="text">
        <string>Relation Name:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="lineEditRelationName"/>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_3">
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="pushButtonOk">
       <property name="text">
        <string>&amp;Ok</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pushButtonCancel">
       <property name="text">
        <string>&amp;Cancel</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer_2">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="mesressources.qrc"/>
 </resources>
 <connections/>
</ui>
